src/lib/y2storage/storage_manager.rb

```diff
@@ -81,6 +81,7 @@ def initialize(storage_environment)
       @storage.activate(activate_callbacks)
       @storage.probe
```

Comment:
What if we simply call the just added `#probe` so don't duplicate responsibilities (like calling `Storage.probe` and updating `@staging_revision_after_probing`)?

I mean
```ruby
  ...
  @storage.activate(activate_callbacks)
  @staging_revision = 0
  @proposal = nil
  probe
end
```